addons/udes_delivery_slot/models/delivery_slot.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models
+
+import datetime
+import math
+from odoo.tools.float_utils import float_round
+
+
+def float_to_time(float_hour):
+    if float_hour == 24.0:
+        return datetime.time.max
+    return datetime.time(int(math.modf(float_hour)[1]),
+                         int(float_round(60 * math.modf(float_hour)[0], precision_digits=0)), 0)
```

Comment:
I thought you'd mentioned an existing Odoo help function that already did this (or a similar) calculation?